### PR TITLE
Feature/cdh 1 homepage and navigation

### PR DESCRIPTION
**Pull request by sarahframe**
in this PR:
- primary navigation
- secondary navigation
- footer
- homepage hero
- cta block - this was me testing so the block is not ready to be reviewed yet

Still need to tidy up existing snippets that we are not using.

### CODE REVIEW

File: cdhweb/pages/mixin.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel, TitleFieldPanel
+
+
+class HomePageHeroMixin(models.Model):
+    title = models.CharField(
```
Comment (haydngreatnews) [inline cdhweb/pages/mixin.py]: If you're going to add this to a `page` model, then there will already be a field called `title`, and this likely won't work -- If you need to adjust the max length, you can do it with the[ LengthOverrideWidget](https://github.com/springload/cookiecutter-wagtail/blob/7352085803d5e03ecfc1a64881a3d2fd1a87ba8e/%7B%7Bcookiecutter.repo_name%7D%7D/core/utils/fields.py#L6) (though maybe try just passing it into the TitleFieldPanel first -- I'm pretty sure you can pass in `help_text` and stuff direct to the Panel now, but not sure about the `max_length`)

File: cdhweb/pages/migrations/0004_homepage_hero_image_homepage_summary_and_more.py [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+import django.db.models.deletion
+import wagtail.blocks
+import wagtail.documents.blocks
+import wagtail.embeds.blocks
+import wagtail.fields
+import wagtail.images.blocks
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (haydngreatnews) [inline cdhweb/pages/migrations/0004_homepage_hero_image_homepage_summary_and_more.py:12]: It looks like this migration is adding the HomePageHeroMixin to the Homepage, but I can't see that change in this PR anywhere?

File: cdhweb/pages/mixin.py (hunk 1 above)
Comment (sarahframe) [reply cdhweb/pages/mixin.py]: thanks - have updated PR

File: cdhweb/pages/migrations/0004_homepage_hero_image_homepage_summary_and_more.py (hunk 2 above)
Comment (sarahframe) [reply cdhweb/pages/migrations/0004_homepage_hero_image_homepage_summary_and_more.py:12]: sorry forgot to add it - there now

**Review by haydngreatnews — APPROVED** [review]

**Pull request merged**